pkg/config/config.go

```diff
@@ -364,8 +365,26 @@ type RemoteWrite struct {
 }
 
 type RemoteWriteTarget struct {
-	Address   string            `desc:"server that implements the pyroscope /ingest endpoint" mapstructure:"address"`
-	AuthToken string            `desc:"authorization token used to upload profiling data" mapstructure:"auth-token"`
-	Tags      map[string]string `name:"tag" desc:"tag in key=value form. The flag may be specified multiple times" mapstructure:"tags"`
-	Timeout   time.Duration     `def:"30s" desc:"profile upload timeout" mapstructure:"timeout"`
+	Address string `desc:"server that implements the pyroscope /ingest endpoint" mapstructure:"address"`
+	// TODO(eh-am): use a custom type here to not accidentaly leak the AuthToken?
+	AuthToken    string            `desc:"authorization token used to upload profiling data" yaml:"auth-token"`
```

Comment:
Huh, this is relevant: https://github.com/pyroscope-io/pyroscope/pull/948

Yeah, may be a custom yaml marshaller implementation can do the trick